charts/linkerd2-cni/values.yaml

```diff
@@ -43,7 +43,9 @@ privileged: false
 # -|- Tolerations section, See the
 # [K8S documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
 # for more information
-#tolerations:
+tolerations:
+- operator: Exists
+  effect: NoSchedule
```

Comment:
The other special case is "An empty effect matches all effects with key `key1`." The first case appears before this one though, so I assume that would take precedence. I also think we'd be fine removing `effect`, but it's not as clear as it could be.

Reply:
I removed the effect and left the Exists operator in and cni is scheduled properly on the node so we don't need an effect. great!